src/RelayClient.ts

```diff
@@ -956,3 +1003,21 @@ export function _dumpRelayingResult(relayingResult: RelayingResult): string {
     }
     return str;
 }
+
+/*
+ * For the moment we hard-coded the trif-rbtc conversion (extracted from the original dapp)
+ */
+function costInWei(maxPossibleGas: BN, gasPrice: BN) {
+    return maxPossibleGas.mul(gasPrice);
+}
+
+function getTRifWei(costInWei: BN) {
+    const tRifPriceInRBTC = 0.000005739;
+    const ritTokenDecimals = 18;
+
+    const costInRBTC = web3.utils.fromWei(costInWei.toString());
+    const costInTrif = parseFloat(costInRBTC) / tRifPriceInRBTC;
+    const costInTrifFixed = costInTrif.toFixed(ritTokenDecimals);
+    const costInTrifAsWei = web3.utils.toWei(costInTrifFixed);
+    return costInTrifAsWei;
+}
```

Comment:
i know it is temporary and hard-coded, but i would move this to a separate file for clearness' sake